tests/controllers/forms.js

```diff
@@ -0,0 +1,119 @@
+/* jshint unused: false */
```

Comment:
Well I thought the tests were readable even if the params in the callback were unused.